apps/app_rand.c

```diff
@@ -84,7 +85,8 @@ int opt_rand(int opt)
         return loadfiles(opt_arg());
         break;
     case OPT_R_WRITERAND:
-        save_rand_file = opt_arg();
+        OPENSSL_free(save_rand_file);
```

Comment:
But you can start openssl without arguments and invoke "apps" or one several times, right?